src/endpoints/schemas/responses/titles.ts

```diff
@@ -0,0 +1,15 @@
+export namespace Schema_1970_01_01 {
+  /** {@link https://wiki.guildwars2.com/wiki/API:2/build} */
+  export interface Title {
+    /** The id of the title. */
+    id: number
+    /** The name of the title (this is also the title displayed over a character in-game.) */
+    name: string
+    /** (Now depreciated) - The id of the achievement that grants this title; resolvable against v2/achievements. */
+    achievement: number
```

Comment:
You can tag deprecated fields with JSDoc, this way IDEs like VSCode can give them extra styling

```ts
/**
* The id of the achievement that grants this title; resolvable against v2/achievements.
 * @deprecated Use "achievements" instead
 */
```

Reply:
Good catch! I blindly copied them over from the wiki, but I should probably do at least one more pass over the docstrings and see if they can be made more explicit by JSDoc tags. 👍